support/reftest/reftest

```diff
@@ -0,0 +1,296 @@
+#!/usr/bin/env python
+"""
+
+Prepare reference test data for Exodus-lambda integration test.
+It requires a "data.yml" file specifying the reference test data.
+
+"""
+
+import sys
+import yaml
+import boto3
+import tempfile
+import requests
+import argparse
+
+from tqdm import tqdm
+from datetime import datetime
+
+
+class DBHandler:
+    def __init__(
+        self,
+        table,
+        access_id=None,
+        access_key=None,
+        session_token=None,
+        region_name=None,
+    ):
+        self.dynamodb = boto3.Session(
+            aws_access_key_id=access_id,
+            aws_secret_access_key=access_key,
+            aws_session_token=session_token,
+            region_name=region_name,
+        ).client("dynamodb")
```

Comment:
Since both this and the s3 client are always used, it would probably be better to only create the session once and then create the client in each *Handler class.
So, instead of passing access_id, access_key, session_token, and region_name to each handler, create the session in main() and pass that object to the handlers.